src/test/render.tsx

```diff
@@ -28,16 +31,64 @@ type CreateWrapperProps = {
   wrapper?: (props: ComponentWrapperProps) => ReactElement;
 };
 
-const DefaultWrapper = ({ children, history, route, meta }: ComponentWrapperProps) => {
+const APP_ROOT = '/a/grafana-synthetic-monitoring-app';
+
+function getRelativeRoute(route?: string) {
+  if (!route) {
+    return '*';
+  }
+  if (route.includes(`${APP_ROOT}/`)) {
+    return route.replace(`${APP_ROOT}/`, '');
+  }
+
+  return route;
+}
+
+const DefaultWrapper = ({ children, route: _route, history, meta }: ComponentWrapperProps) => {
+  const route = getRelativeRoute(_route);
+
   return (
     <QueryClientProvider client={getQueryClient()}>
       <MetaContextProvider meta={{ ...SM_META, ...meta }}>
         <FeatureFlagProvider>
           <SMDatasourceProvider>
             <PermissionsContextProvider>
-              <Router history={history}>
-                <Route path={route}>{children}</Route>
-              </Router>
+              <MemoryRouter initialEntries={history.entries}>
+                <CompatRouter>
+                  <Routes>
+                    <Route path={APP_ROOT}>
+                      <Route
+                        path={route}
+                        element={
+                          <>
+                            <TestRouteInfo />
```

Comment:
Can we document what this is doing, please.  It took me a while to work out its purpose and had to look at some of the tests. Would help me out on getting started on this ticket, too: https://github.com/grafana/synthetic-monitoring-app/issues/875 🙏

Reply:
I added some JSDoc to the component.
Depending on your IDE, it should allow for some additional clarification

**Using JetBrains (GoLand)**
![image](https://github.com/user-attachments/assets/f8ceed4a-0174-4e7f-a777-076e0ad3a3f4)

Reply:
![A screenshot of the JSDoc info for TestRouteInfo in VSCode.](https://github.com/user-attachments/assets/04d3e752-3f39-4628-b2c0-02fe3cc19af6)

📜🥳 🎉